addons/website_forum/models/forum.py

```diff
@@ -414,7 +414,7 @@ def _check_post_type(self):
         if (self.post_type == 'question' and not self.forum_id.allow_question) \
                 or (self.post_type == 'discussion' and not self.forum_id.allow_discussion) \
                 or (self.post_type == 'link' and not self.forum_id.allow_link):
-            raise UserError(_('This forum does not allow %s') % self.post_type)
+            raise ValidationError(_('This forum does not allow %s' % self.post_type))
```

Comment:
I think that it will be difficult to export the translation that way. @mart-e can you confirm ?